mobile_back_tflite/cpp/backend_tflite/tflite_settings_apple.h

```diff
@@ -31,36 +31,36 @@ common_setting {
 
 benchmark_setting {
   benchmark_id: "image_classification"
-  accelerator: "coreml"
-  accelerator_desc: "CoreML"
-  configuration: "TFLite"
+  accelerator: "ane"
+  accelerator_desc: "ANE"
+  framework: "TFLite CoreML"
   model_path: "https://github.com/mlcommons/mobile_models/raw/main/v1_1/tflite/mobilenet_edgetpu_224_1.0_float.tflite"
   model_checksum: "66bb4eba50987221608f8487ed405794"
 }
 
 benchmark_setting {
   benchmark_id: "object_detection"
-  accelerator: "coreml"
-  accelerator_desc: "CoreML"
-  configuration: "TFLite"
+  accelerator: "ane"
+  accelerator_desc: "ANE"
+  framework: "TFLite CoreML"
   model_path: "https://github.com/mlcommons/mobile_models/raw/main/v1_0/tflite/mobiledet.tflite"
   model_checksum: "566ceb72a4c7c8926fe4ac8eededb5bf"
 }
 
 benchmark_setting {
   benchmark_id: "natural_language_processing"
-  accelerator: "metal"
-  accelerator_desc: "Metal"
-  configuration: "TFLite"
+  accelerator: "gpu"
+  accelerator_desc: "GPU"
+  framework: "TFLite Metal"
```

Comment:
You should also update the `metal` check in tflite backend to match new settings, or revert it.

By the way, `gpu` value is already used, and it's different from `metal`.

Reply:
Done.